### PR TITLE
Added IO timing graph to FilesystemDetails dashboard

### PR DESCRIPTION
**Pull request by keithf4**

### CODE REVIEW

File: grafana/FilesystemDetails.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "__inputs": [
     {
       "name": "DS_PROMETHEUS",
-      "label": "PROMETHEUS",
+      "label": "Prometheus",
```
Comment (sharmay) [inline grafana/FilesystemDetails.json]: I see that DS Label keeps changing between upper case and lower case. Please plan to keep it consistent.

File: grafana/FilesystemDetails.json [hunk 2]
```diff
@@ -472,5 +559,5 @@
   },
   "timezone": "",
   "title": "FilesystemDetails",
-  "version": 8
+  "version": 3
```
Comment (sharmay) [inline grafana/FilesystemDetails.json]: For version lets keep it going up not down.

Comment (keithf4) [reply grafana/FilesystemDetails.json]: This metric was already supplied by node_exporter

The version number is going to keep resetting every time anyone makes a new installation or someone different contributes a dashboard. Can't really control it.

Comment (sharmay) [reply grafana/FilesystemDetails.json]: But we can force it to increment

File: grafana/FilesystemDetails.json (hunk 1 above)
Comment (sharmay) [reply grafana/FilesystemDetails.json]: Any comment on DS Label ?

**Comment by keithf4** [comment]
Both the issues you rose will likely be handled better by using the provisioning system in 5.x.

**Comment by keithf4** [comment]
Metric name used in this graph was changed in 0.16.0. Closing this pull request and will include this metric with new name in new provisioned version of FilesystemDetails dashboard in upcoming PR.

**Pull request closed without merging**